app/assets/javascripts/footer.js

```diff
@@ -8,7 +8,7 @@ bottom = function () {
     var footerPadding = parseInt($footer.css('padding-top')) +
                         parseInt($footer.css('padding-bottom'));
 
-    var diff = docHeight - copyrightBottom + footerPadding;
+    var diff = docHeight - copyrightBottom + footerPadding - 28;
```

Comment:
> This wasn't working anymore? :/

No, it has not worked for a long time. I'm not sure where 28 actually comes from, but it fixes the problem